### PR TITLE
learnt useState coding

### PR DESCRIPTION
**Pull request by komal7292**

### CODE REVIEW

File: komal/Learnt-useState/src/reportWebVitals.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+const reportWebVitals = onPerfEntry => {
```
Comment (rohitksp) [inline komal/Learnt-useState/src/reportWebVitals.js:1]: Can you please tell me why we need this function?

**Review by rohitksp — CHANGES_REQUESTED** [review]

**Review by rohitksp — COMMENTED** [review]
Spelling mistake "useSate" -> "useState"

File: komal/Learnt-useState/src/App.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import React, { useState } from 'react'
+
+const App = () => {
+  const state = useState()
+
+  const [Count, setCount] = useState(0)
```
Comment (nrchandan) [inline komal/Learnt-useState/src/App.js]:
```suggestion
  const [count, setCount] = useState(0)
```
Variable names should start with lowercase.

File: komal/Learnt-useState/src/App.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+  const IncNum = () => {
```
Comment (nrchandan) [inline komal/Learnt-useState/src/App.js]:
```suggestion
  const incNum = () => {
```
Function names should also begin with a lowercase letter.

**Review by nrchandan — CHANGES_REQUESTED** [review]
I left a couple of comments. Also, I suggest that you follow a similar naming convention for the folders that the instructor follows i.e. name it after the thing we are building, not what we are learning. There may be multiple learnings in a project.

File: komal/Learnt-useState/src/App.js (hunk 3 above)
Comment (komal7292) [reply komal/Learnt-useState/src/App.js]: okay

**Review by nrchandan — APPROVED** [review]
Looks good. Please merge it.

**Review by rohitksp — APPROVED** [review]

**Pull request merged**